test/jdk/java/awt/Frame/FrameStateTest/FrameStateTest.java

```diff
@@ -175,13 +170,13 @@ static class CreateFrame extends Frame
                     ((isResizable) ? "RESIZABLE" : "NON-RESIZABLE"));
 
             setLayout(new FlowLayout());
-            add(b1 = new Button("resizable"));
-            add(b2 = new Button("resize"));
-            add(b3 = new Button("iconify"));
-            add(b4 = new Button("iconify and restore"));
-            add(b5 = new Button("hide and show"));
-            add(b6 = new Button("hide, iconify and show"));
-            add(b7 = new Button("hide, iconify, show, and restore"));
+            add(b1 = new Button("Resizable"));
+            add(b2 = new Button("Resize"));
+            add(b3 = new Button("Iconify"));
+            add(b4 = new Button("Iconify and Restore"));
+            add(b5 = new Button("Hide and show"));
```

Comment:
To be consistent with other buttons , this button should can be changed from  text "Hide and show"   to "Hide and show"